libs/vkd3d/command.c

```diff
@@ -5419,6 +5421,26 @@ static void d3d12_command_list_init_default_descriptor_buffers(struct d3d12_comm
     }
 }
 
+static void d3d12_command_list_reset_rtv_resolves(struct d3d12_command_list *list)
+{
+    unsigned int i;
+
+    for (i = 0; i < list->rtv_resolve_count; i++)
+    {
+        struct d3d12_rtv_resolve *resolve = &list->rtv_resolves[i];
+
+        resolve->src_resource = NULL;
```

Comment:
Any reason to clear out memory like this?

Reply:
This used to store an array locally but I guess this can just be left alone now.

Reply:
Removed this bit, the array does not need to be initialized at all anymore.